clients/native/examples/js-examples/websocket/src/index.js

```diff
@@ -41,7 +41,8 @@ function handleResponse(resp) {
             ourAddress = response.address;
             display("Our address is:  " + ourAddress + ", we will now send messages to ourself.");
         } else if (response.type == "received") {
-            handleReceivedTextMessage(response)
+            // handleReceivedTextMessage(response)
```

Comment:
What happened to `handleReceivedTextMessage` function? If it's unused, just remove it